src/reducers/filter.js

```diff
@@ -0,0 +1,11 @@
+export default (state = {}, action) => {
+  switch (action.type) {
+    case ('CHANGE_FILTER'):
+      return ({
+        ...state,
+        current: action.payload,
+      });
+    default:
+      return state;
+  }
```

Comment:
- It should default to `All`  see [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone4/HandelChangeFilter.png) from Pathwright